src/shared/cloud_limits.c

```diff
@@ -0,0 +1,103 @@
+/* Copyright (C) 2022, Wazuh Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+*/
+
+#include "cloud_limits.h"
+
+#ifdef WAZUH_UNIT_TESTING
+#define static
+#undef OSSEC_LIMITS
+#define OSSEC_LIMITS  "./limits.conf"
+#endif
+
+time_t last_mod_date = 0;
+
+// load json objects from limits.conf file.
+
+int load_limits_file(const char *daemon_name, cJSON ** daemon_obj) {
+
+    if (!daemon_name) {
+        mdebug2("Invalid daemon name is null");
+        return LIMITS_NULL_NAME;
+    }
+
+    time_t cur_mod_date = 0;
+    if ((cur_mod_date = File_DateofChange(OSSEC_LIMITS)) == -1) {
+        mdebug2("File %s doesn't found", OSSEC_LIMITS);
+        last_mod_date = 0;
```

Comment:
This is not necessary.

Reply:
stat functionality does not change file creation/modification parameters, so it is necessary clean the last_mod_date value if file was ranamed or moved.